packages/replicate/src/replicate-image-model.ts

```diff
@@ -126,6 +126,20 @@ export class ReplicateImageModel implements ImageModelV3 {
       }
     }
 
+    // Extract maxWaitTimeInSeconds from provider options and prepare the rest for the request body
+    const { maxWaitTimeInSeconds, ...inputOptions } = replicateOptions ?? {};
+
+    // Build the prefer header based on maxWaitTimeInSeconds:
+    // - undefined/null: use default sync wait (prefer: wait)
+    // - 0: disable sync mode (no prefer header)
```

Comment:
is disabling the sync mode supported in our code? if not we should not support it in the header (ie remove that part here)